AUTOTEST/runtests-ij

```diff
@@ -1,2 +1,2 @@
-TEST_ij/*.sh
+TEST_ij/[^b]*.sh
```

Comment:
The "standard" test directories like `TEST_ij`, `TEST_struct`, `TEST_sstruct` should only contain tests that represent "default" tests that are always run on the Linux boxes when testing new code.  I regularly do `runtest.sh TEST_ij/*.sh` to do a quick test of the IJ-related solvers when I am modifying code.  Same for the other two directories.  Once I get these basic tests working, then I usually move on to running the full suite of tests in the `AUTOTEST` directory.

So, I think these new benchmark tests should probably go in a new directory, say `TEST_bench` or something.

Reply:
@rfalgout This makes more sense. Sorry about that. I will change it back. That `.sh` was in `TEST_cuda` and I remembered we talked about directories like it or `TEST_single, TEST_longdouble` could be combined with `TEST_ij` after we had `-save` option in `runtest.sh`. That was why I moved it. But now I can see why I should not have done that. Thanks!